src/jvm/main/org/jetbrains/kotlinx/lincheck/Reporter.kt

```diff
@@ -236,4 +244,12 @@ private fun StringBuilder.appendException(t: Throwable) {
     val sw = StringWriter()
     t.printStackTrace(PrintWriter(sw))
     appendln(sw.toString())
+}
+
+internal fun StringBuilder.appendStateEquivalenceViolationMessage(sequentialSpecification: Class<*>) {
+    append("To verify outome results faster, it is highly recommended to specify the state equivalence relation on your" +
+        "sequential specification. However, on $sequentialSpecification it is is not definied or implemented incorrectly. " +
```

Comment:
"is **is** not **definied** or implemented incorrectly" -> "is not defined or **is** implemented incorrectly"